packages/perseus-editor/src/components/sortable.tsx

```diff
@@ -117,57 +142,75 @@ const SortableArea = createReactClass({
             dragItem.removeEventListener("dragstart", this._listenEvent);
             dragItem.removeEventListener("drop", this._cancelEvent);
         }
-    },
-    render: function () {
-        const sortables = this.state.components.map((component, index) => (
+    }
+
+    render() {
+        const sortables = this.props.components.map((component, index) => (
             <SortableItem
                 index={index}
                 component={component}
                 area={this}
                 key={component.key}
                 draggable={component.props.draggable}
-                dragging={index === this.state.dragging}
+                dragging={index === this.dragging}
             />
         ));
         return (
             <ol className={this.props.className} style={this.props.style}>
                 {sortables}
             </ol>
         );
-    },
-});
+    }
+}
+
+type ItemProps = {
+    area: any;
+    component: any;
```

Comment:
Can we type this as `React.ReactNode`?  As far as I can see there are no constraints on what this is except that it is something React can render.